### PR TITLE
docs: rewrite readme

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Rewrite readme
- Add screenshot of badges

## Context:

Full rewrite of the readme.

## TODO:

- [x] Discuss screenshot style
- [x] Create improved screenshot
- [x] Put screenshot on Renovate server instead of my fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,97 @@
...
-## Confidence Levels and their meaning
+- Prevent updates which break in production
+- See at a glance if you should update
 
-Merge Confidence has four possible values:
+Merge Confidence finds and flags undeclared breaking releases.
+It analyzes test and release adoption data across WhiteSource Renovate’s early-adopting user base.
 
-- **Low**: If an update is classified as having low merge confidence, it means we think it contains breaking changes. Often this is because it's a Major version update so this is intentional, but sometimes it's unintentional or undeclared.
+![Renovate PR with Merge Confidence badges](https://github.com/HonkingGoose/merge-confidence/blob/docs/rewrite-merge-confidence-readme/merge-confidence-badges.png)
```
Comment (HonkingGoose) [inline README.md:11]: We should use a image to sell this feature to our readers. 😉 

Should the image show:

- Only the table with the badges?
- Table with badges on a blurred PR background (Example: [GitHub Blog](https://github.blog/2022-03-21-start-working-on-github-issues-faster/))?
- The full sharp PR (what we have now)

File: README.md [hunk 2]
```diff
@@ -1,69 +1,97 @@
 # Merge Confidence
 
-Merge Confidence identifies and flags undeclared breaking releases based on analysis of test and release adoption data across WhiteSource Renovate’s early-adopting user base.
-The new feature was created to help users avoid the pain of un-mergeable Pull Requests or worse -- a broken dependency in production.
+Look at the Merge Confidence badges before merging to:
```
Comment (HonkingGoose) [inline README.md:3]: Let's put the important stuff for the reader up first, and sell them on this awesome feature. 😄

**Review by HonkingGoose — COMMENTED** [review]
Some comments to get the review started. 😉

File: README.md (hunk 1 above)
Comment (rarkins) [reply README.md:11]: I'd like to see how it looks with blurred and non blurred, if you're interested.

Comment (HonkingGoose) [reply README.md]: @rarkins You can find a new image in the "Files changed" tab of this PR. Let me know what you think! 😄

**Review by rarkins — APPROVED** [review]

**Pull request merged**